.github/workflows/python.yml

```diff
@@ -109,7 +109,7 @@ jobs:
         id: install_deps
         run: |
           python -m pip install --upgrade pip
-          pip install pipenv
+          pip install pipenv==2023.7.4
```

Comment:
This site does not use the keep-outdated flag, updating anyways for consistency.